App/Sagas/HandleImageUploadComplete.ts

```diff
@@ -0,0 +1,38 @@
+import { call, put, select } from 'redux-saga/effects'
```

Comment:
Hey look, not adding any more to `TextileSagas`. Putting new sagas in separate files.

Reply:
yes! about time